src/patterns/molecules/error/error.stories.js

```diff
@@ -47,8 +47,8 @@ export const ErrorAr = () => error(
   {
     attributes:
       {
-        dir: 'rtl',
-        lang: 'ar',
+        'data-storybook-htmlattr-dir': 'rtl',
```

Comment:
Note that this doesn't stop `dir` and `lang` being applied here: these would be applied on the root element of the pattern.